tests/cpu_bugs/xen_domu_mitigation_test.pm

```diff
@@ -131,7 +131,13 @@ my $pti_on_on_hvm = {"pti=on" => {
         }
     }
 };
-
+my $pti_on_on_hvm_for_cascadelake = {"pti=on" => {
+        default => {
+            expected => {'cat /proc/cmdline' => ['pti=on'], 'cat /sys/devices/system/cpu/vulnerabilities/meltdown' => ['Not affected']},
+            unexpected => {'cat /proc/cmdline' => ['pti=off'], 'cat /sys/devices/system/cpu/vulnerabilities/meltdown' => ['Unknown.*XEN PV detected,.*hypervisor mitigation required']}
```

Comment:
I have a stupid question, for these logic, the expected value is fixed, however, is the unexpected value also fixed? or not? So are all below unexpected values logic.

Reply:
unexpected value not fixed. The secnarion is when come across the expection string,If it doesn't appear go die.
when come across the unexpect string,if it appears go die. And the default match is the expect. So I did not fix the unexpected value .